.github/workflows/tox.yml

```
@@ -50,6 +50,28 @@ jobs:
         name: code-coverage-report
         path: tests/_reports/
 
+  tox-quickstart-guide:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.10"]
```

Since there is only one configuration, it's cleaner as env vars:
```suggestion
    env:
      PY_VERSION: "3.10"
```

---

Okay, let's leave it as a matrix then.